File: algorithm/src/main/java/baekjoon/p02675/Main.java
package baekjoon.p02675;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();

        for (int i = 0; i < t; i++) {
            int r = sc.nextInt();
            String s = sc.next();

            String result = solve(r, s);
            System.out.println(result);
        }
    }

    private static String solve(int r, String s) {
        StringBuilder sb = new StringBuilder();

        for (int i = 0; i < s.length(); i++) {
            for (int j = 0; j < r; j++) {
                char ch = s.charAt(i);
                sb.append(ch);
            }
        }

        return sb.toString();
    }

}

File: algorithm/src/main/java/baekjoon/p02675/Main.kt
package baekjoon.p02675

fun main(args: Array<String>) {
    repeat(readLine()!!.toInt()) {
        val input = readLine().toString()
        val inputs = input.split(" ")

        val r = inputs[0].toInt()
        val s = inputs[1]
        println(solve(r, s))
    }
}

private fun solve(r: Int, s: String): String {
    val sb = StringBuilder()

    for (ch in s.drop(0)) {
        repeat(r) {
            sb.append(ch)
        }
    }

    return sb.toString()
}